pkg/exec/builder/execute_test.go

```diff
@@ -134,3 +135,22 @@ func TestExecuteStep_HasOrderedArguments(t *testing.T) {
 	_, err := ExecuteStep(c.Context, step)
 	require.NoError(t, err, "ExecuteStep failed")
 }
+
+func TestExecuteStep_SpecifiesCustomWorkingDirectory(t *testing.T) {
+	c := context.NewTestContext(t)
+	wd := os.TempDir()
+
+	step := TestOrderedStep{
+		TestStep: TestStep{
+			Command:          "uname",
```

Comment:
Let's have the command return us the current working directory so that we know that it was set properly
```suggestion
			Command:          "pwd",
```